src/lib/server.ts

```diff
@@ -4,7 +4,6 @@ import * as http from 'http';
 // @ts-ignore
 import * as destroyable from 'server-destroy';
 import { bindExpress as bindSentry, SentrySettings } from './monitoring/sentry';
-import override from './util/override';
```

Comment:
i think `override` is unused right? we can remove it

Reply:
True. File removed :fire: